src/visitors/defuse_analyze_visitor.hpp

```diff
@@ -237,71 +234,34 @@ class DefUseAnalyzeVisitor: public AstVisitor {
     /// unsupported statements : we aren't sure how to handle this "yet" and
     /// hence variables used in any of the below statements are handled separately
 
-    void visit_reaction_statement(ast::ReactionStatement& node) override {
-        visit_unsupported_node(&node);
-    }
-
-    void visit_non_lin_equation(ast::NonLinEquation& node) override {
-        visit_unsupported_node(&node);
-    }
-
-    void visit_lin_equation(ast::LinEquation& node) override {
-        visit_unsupported_node(&node);
-    }
-
```

Comment:
Same as perf_visitor.hpp - does it make sense to move out one liner function if those header are not used out-of-their-own-compilation-units?